tutorials/single-page-tutorials/nested-nfts-with-wallet-lib.md

```diff
@@ -0,0 +1,423 @@
+## Storyline
+
+Let's imagine a game with avatars and items, which all are represented as unique NFTs on the Shimmer network. In this example it allows an avatar to not only hold items in-game, but its avatar NFT to also own its unique item NFTs directly on-DLT, in order to represent the dependencies between in-game components on the Shimmer network.
+
+![Blue Warrior](./images/warrior-blue.gif)
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/).
+- This tutorial will build on the tutorial [Testnet Wallet Setup](https://wiki.iota.org/tutorials/wallet-setup). If you haven't completed it, please do so, and afterward, come back here, or make sure you have a prefunded wallet. You will use the same folder and development environment as in the mentioned prerequired tutorial, allowing you to reuse the Stronghold account with funds on it.
+
+## Set Up
+
+### Prepare Your Development Environment
+
+Navigate to the folder you created when completing the [Testnet Wallet Setup](https://wiki.iota.org/tutorials/wallet-setup):
+
+```bash
+cd wallet-setup
+```
+
+#### Install Packages
+
+Next to the existing dependencies of the wallet setup tutorial, you will only need to add two more packages by running the following commands:
+
+```bash
+npm install @iota/client
+npm install ipfs-core@0.10.8
+```
+
+Afterward, your `package.json` file should contain the following dependencies:
+
+```json {6-12}
+{
+  "name": "wallet-setup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "create-mnemonic.js",
+  "dependencies": {
+    "@iota/client": "^3.0.0-alpha.7",
+    "@iota/wallet": "^2.0.2-alpha.21",
+    "bip39": "^3.0.4",
+    "dotenv": "^16.0.1",
+    "ipfs-core": "^0.15.4"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0"
+}
+```
+
+---
+
+### Prepare Images for your nested NFTs
+
+Add three images to your folder and make sure they are named correctly with `avatar.jpg`, `mask.jpg` and `weapon.jpg`. You can use the images below, or pick your own images as desired:
+
+![Warrior avatar](./images/warrior-avatar.png)
+![Warrior mask](./images/warrior-mask.png)
+![Warrior weapon](./images/warrior-weapon.png)
+
+---
+
+## Scripts
+
+### Create NFTs
+
+Create a new file `create-nfts.js` and add the following code.
+
+:::note
+
+We broke the code into separate snippets to help you understand it better. To make it work, copy all code snippets one after another into the file that you have just created.
+
+:::
+
+#### 1. Imports and parameters
+
+```javascript
+// Libraries
+const { utf8ToHex } = require('@iota/client');
+const { AccountManager } = require('@iota/wallet');
+const IPFS = require('ipfs-core');
+const fs = require('fs');
+
+// Environment variables
+require('dotenv').config({ path: './.env' });
+const password = process.env.SH_PASSWORD;
+const accountName = process.env.ACCOUNT_NAME;
+
+// Network configuration
+const { networkConfig } = require('./networkConfig.js');
```

Comment:
If we change the `networkConfig.js` module to just export the object directly, you don't need to unpack the export object.

```suggestion
const networkConfig = require('./networkConfig.js');
```

Reply:
Agreed. But since this has dependencies to the wallet setup tutorial, I'll keep it this way to avoid any breaks until we've aligned on if we want any tutorial to be independent, or if they are connected learning paths.